packages/meditrak-app-server/src/app/createApp.ts

```diff
@@ -25,6 +28,7 @@ export function createApp(database = new TupaiaDatabase()) {
   const authMiddleware = buildAuthMiddleware(database);
   const app = new MicroServiceApiBuilder(database)
     .attachApiClientToContext(authHandlerProvider)
+    .use('*', checkAppVersion)
```

Comment:
This enforces that the `appVersion` parameter is always passed in for all routes